README.md

```diff
@@ -105,6 +105,9 @@ The default configuration provisions ElasticMQ with a initial queue of the same
 
 To make use of this queue, point your client to: `http://localhost:9324/queue/default`.
 
+#### Dead letter queue
+The default dead letter queue is called dlq, configured in opt/elasticmq.conf
```

Comment:
Maybe make a reference to: https://github.com/softwaremill/elasticmq#automatically-creating-queues-on-startup

Reply:
Done